neuro_san/internals/run_context/langchain/llms/default_llm_factory.py

```diff
@@ -178,8 +171,7 @@ def create_llm(
                 unknown to this method.
         """
         full_config: Dict[str, Any] = self.create_full_llm_config(config)
-        llm_client: LangChainLlmClient = self.create_llm_client(full_config)
-        llm_resources: LangChainLlmResources = self.create_llm_resources_with_client(full_config, llm_client)
+        llm_resources: LangChainLlmResources = self.create_llm_resources(full_config)
```

Comment:
Abandon the LangChainLlmClient and factory idea.
That said, we still need a new LlmFactory method that properly returns us LlmResources.